tasks/xpack/security/elasticsearch-xpack-activation.yml

```diff
@@ -27,7 +27,7 @@
     return_content: yes
   register: license_activated
   no_log: True
-  when: es_xpack_features is defined and "'security' in es_xpack_features"
+  when: es_api_basic_auth_username is defined or es_api_basic_auth_password is defined
```

Comment:
I think this should be an `and` rather than `or` for the edge case where someone has only defined the username or password. However I believe at the start of the playbook we detect that both are defined when security is enabled, so things should fail earlier anyway.